README.md

```diff
@@ -1,70 +1,69 @@
-# Getting Started with Create React App
+# metrics-webapp
```

Comment:
- [ ] [REQUIRED] Kindly ensure that you record a video presenting the features of the project with a maximum length of 5 minutes. The video should contain a clear description of the project,a demo of the project, highlighting all the project features, and also the video should contain a highlight about a piece of the code you are proud of. This is a core requirement in this project.